htlcswitch/link.go

```diff
@@ -2271,6 +2258,19 @@ func (l *channelLink) HtlcSatifiesPolicyLocal(payHash [32]byte,
 	)
 }
 
+// createFailureWithUpdate retrieves this link's last channel update message and
```

Comment:
Should be moved in the file to lie before it's used.

Reply:
moved up